test/lib/test-cluster.js

```diff
@@ -133,7 +133,7 @@ function TestCluster(opts) {
     self.timers = null; // Set whenever a channel is created
 
     self.tchannelJSON = TChannelJSON();
-    self.logger = DebugLogtron('autobahn');
+    self.logger = opts.logger || DebugLogtron('autobahn');
```

Comment:
you can also do

```
DebugLogtron('autobahn', {
    enabled: opts.loggerEnabled
});
```

Setting `enabled` to false, silences everything except `error` and `fatal`.